Tests/CommandsTests/TestToolTests.swift

```diff
@@ -266,6 +266,19 @@ final class TestToolTests: CommandsTestCase {
         }
     }
 
+    // TODO: This test should be moved into `ResourceTests.swift` in the
+    // `FunctionalTests` scheme when the `FunctionalTests` scheme is re-enabled.
+    func testResourcesInMixedClangPackage() throws {
+        #if !os(macOS)
+        // Running swift-test fixtures on linux is not yet possible.
+        try XCTSkipIf(true, "test is only supported on macOS")
+        #endif
+
+        try fixture(name: "Resources/Simple") { fixturePath in
+            XCTAssertBuilds(fixturePath, extraArgs: ["--target", "MixedClangResource"])
+        }
+    }
```

Comment:
Per https://github.com/apple/swift-package-manager/pull/5921#discussion_r1033909217, I'll move this to the `ResourceTests.swift` and then this PR is gtg.

Reply:
Done in 25b9bf3. 

cc @neonichu